seagence/README.md

```diff
@@ -0,0 +1,56 @@
+# Seagence
+
+## Overview
+
+[Seagence][1] is a defect monitoring platform for Java applications. Using ExecutionPath technology (aka GPS Tracker for transactions), Seagence detects unknown defects caused by various issues like swallowed exceptions, multithreading issues, and others including defects that disguise in a 200 success HTTP response code.
+
+Seagence integrates with Datadog using an API-based integration. Seagence's tiny Java agent is attached to your production application and posts collected data (ExecutionPaths, errors/exceptions, and contextual information) to the Seagence backend. Continuously analyzing the data stream from Seagence's Java agent, Seagence's backend, using ExecutionPath technology, detects defects when they occur, including their root cause, in real-time. This eliminates the need for debugging and troubleshooting. Using a Rest API, Seagence then posts detected defects and their root cause as Datadog Events to the Datadog backend. You can find the details of the defects in the "Seagence - Defects Overview" dashboard or you can set up a Datadog Monitor to trigger alerts. More details about defects can be found on [SeagenceWeb][2]. With a Seagence provided defect and the root cause in hand, you can fix your broken code.
+
+### Usage
+
+The Seagence integration comes with a dashboard called "Seagence - Defects Overview". The top widget shows the defects timeline and the bottom widget shows a list of defects including their root cause exception with complete stack trace. In the defects timeline view, defects are shown as vertical red bars. You can click any vertical bar to open the context menu. Click "View related events" to open the defects as Datadog Events in the "Events Explorer" dashboard.
+
+## Setup
+
+### Installation
+
+Visit [Seagence][1] to sign up for free. Once registered, you can initiate the connection between Seagence and Datadog in 2 ways:
+1. Find the Seagence tile in Datadog's Integrations view and click **Install Integration**. Click **Connect** on the tile, which guides you through the Datadog OAuth2 flow to grant Seagence access to post Events to your Datadog account.
+2. Alternatively, log in into your Seagence account and go to **Settings** > **Integrations**. Add a Datadog connection and follow the prompts to set up OAuth2 and grant Seagence access to post Events to your Datadog account.
```

Comment:
We do not recommend prompting users to initiate authorization from outside of the Datadog tile. Within Datadog, we can block users from authorizing if they do not have the sufficient permissions that are being requested. However, if authorization is initiated from outside of Datadog, we are unable to block that and users with insufficient permissions may authorize the integration, which will lead to errors from our API.

Reply:
Removed instruction to initiate the connection from Seagence.